website/docs/docs/building-a-dbt-project/package-management.md

```diff
@@ -168,3 +168,16 @@ packages:
 ```
 
 </File>
+
+### Setting two-part versions
+As of 0.17.0, if the package version you want is only specified as `major`.`minor`, as opposed to `major.minor.patch`, you may get an error that `1.0 is not of type 'string'`. In that case you will have to tell dbt that your version number is a string.
+
+<File name='packages.yml'>
+ 
```

Comment:
this breaks my brain, but adding another newline here should make make this work